metatron-app/metatron-discovery
# #3969 Fixed an issue where labels in long merge fields were not visible

### Description
There is an error where labels for long merge fields are not visible in grid charts.

**Related Issue** : #3969 


### How Has This Been Tested?
1. Create a grid chart that merges multiple fields.
2. Check that the field values in the grid are displayed normally.
3. Check if the field value is displayed even when scrolling.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
